app/resources/assets/sass/_profile.scss

```diff
@@ -49,7 +49,14 @@
 			display: none;
 		}
 	}
-
+	
+ select.form-control{
+    	background: url(https://i.stack.imgur.com/tLwVk.png) right / 22px no-repeat #fff;
```

Comment:
If you include images, it would be good to copy them into the hhaccessibility.github.io\app\public\images directory and access them from there.  A few reasons for this include us controlling how available the image is and the image stays available when developing without Internet service.